test/unit/compileProtos.ts

```diff
@@ -205,4 +205,19 @@ describe('compileProtos tool', () => {
     ]);
     assert.strictEqual(rootName, 'default');
   });
+
+  it('reformat the JSDOC link in the JS and TS file', async () => {
+    await compileProtos.main([
+      path.join(__dirname, '..', '..', 'test', 'fixtures', 'protoLists'),
+    ]);
+    assert(fs.existsSync(expectedJSResultFile));
+    assert(fs.existsSync(expectedTSResultFile));
+    const js = await readFile(expectedJSResultFile);
+    const ts = await readFile(expectedTSResultFile);
+    // TODO: use assert.match and assert.doesnotmatch after node version > 13
+    assert.ok(/{@link (.*?)|(.*?)}/.test(js.toString()));
```

Comment:
This regex does not work as you expect - you need to quote `|` with a backslash, otherwise it's a special character.

Reply:
Actually, instead of looking at regex match, I would check for a known substring (since we know exactly which proto file is used as a test input).

Reply:
A good idea. Update the tests. Thank you.